src/TouchCarousel.js

```diff
@@ -134,9 +160,21 @@ class TouchCarousel extends React.PureComponent {
     }
   }
 
+  onDocumentMouseUp = (e) => {
+    if (!this.isMouseDown) {
+      return
+    }
+    this.isMouseDown = false
+    this.onEndOrCancel(1, e)
+  }
```

Comment:
Does desktop has something like the `touchCancel` event? What happens if the user switches the browser tab or minimizes the browser window during dragging?

Reply:
Good point, the correct event is `blur`. I added it to the event listeners.